src/RSocketSetup.h

```diff
@@ -22,7 +24,7 @@ class RSocketStats;
 class RSocketSetup {
  public:
   RSocketSetup(
-      std::shared_ptr<FrameTransport> frameTransport,
+      yarpl::Reference<FrameTransport> frameTransport,
```

Comment:
Is there any benefit of changing this? It just seems risky and makes all users have to use custom types instead of a standard shared_ptr.